homeassistant/components/spotify/browse_media.py

```diff
@@ -151,9 +181,20 @@ async def async_browse_media(
     can_play_artist: bool = True,
 ) -> BrowseMedia:
     """Browse Spotify media."""
-    if not (info := next(iter(hass.data[DOMAIN].values()), None)):
-        raise BrowseError("No Spotify accounts available")
-    return await async_browse_media_internal(
+    info = None
+    parsed_url = urlparse(media_content_id)
```

Comment:
Why not use `yarl.URL`?

Also, let's guard it starts with spotify_prefix.

Reply:
Fixed in 5d06ed7